src/Common/MoistThermodynamics/states.jl

```diff
@@ -143,17 +161,19 @@ Constructs a [`PhaseEquil`](@ref) thermodynamic state from:
  - `θ_liq_ice` liquid-ice potential temperature
  - `ρ` (moist-)air density
  - `q_tot` total specific humidity
- - `tol` tolerance for saturation adjustment
+ - `temperature_tol` estimated temperature tolerance for saturation adjustment
  - `maxiter` maximum iterations for saturation adjustment
 """
 function LiquidIcePotTempSHumEquil(
     param_set::APS,
     θ_liq_ice::FT,
     ρ::FT,
     q_tot::FT,
-    maxiter::Int = 30,
-    tol::FT = FT(1e-1),
+    maxiter::Int = 40,
+    temperature_tol::FT = FT(1e-5),
```

Comment:
This seems unreasonably small. Why? We will never know temperature to this accuracy. Better:
```suggestion
    temperature_tol::FT = FT(1e-2),
```

Reply:
I updated this to `temperature_tol=1e-1`, and there, the best we can do is `maxiter=36`